packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -90,4 +123,67 @@ describe("<DatePicker />", () => {
 
     expect(button).toHaveFocus()
   })
+
+  describe("Validation", () => {
+    it("displays error message when status is error", async () => {
+      render(
+        <DatePicker
+          {...defaultProps}
+          status="error"
+          validationMessage="Invalid Date."
+        />
+      )
+
+      expect(screen.getByText("Invalid Date.")).toBeInTheDocument()
+    })
+
+    it("displays error message when selected day is invalid", async () => {
```

Comment:
This (and below) can go in their own `describe` (it can live within the Validation one) since it's tests about inbuilt validation, whereas the one above is about custom validation.